spec/requests/portfolio_items_spec.rb

```diff
@@ -330,14 +330,15 @@
     end
 
     context "when copying into a different portfolio in a different tenant" do
-      let(:params) { { :portfolio_id => not_my_portfolio.id } }
+      let(:params) { { :portfolio_id => not_my_portfolio.id.to_s } }
       let(:not_my_portfolio) { create(:portfolio, :tenant => create(:tenant, :external_tenant => "xyz")) }
 
       before do
         copy_portfolio_item
       end
 
       it 'returns a 422' do
+        response.body
```

Comment:
What change required this addition?

Reply:
@syncrou that was something I was testing with , removed it